### PR TITLE
docs(file-upload): Fix typo Multiple Fields.

### PR DESCRIPTION
**Pull request by Flightkick**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?
Typo in header, I believe `Multiple files` should be `Multiple fields`.
Multiple files topic is already handled before this section.

## What is the new behavior?
- Header now displays `Multiple fields`.

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

Header change so existing hyperlinks might break.
Referees should update their references.

### CODE REVIEW

**Comment by Flightkick** [comment]
I just think of something, maybe it is possible to put an empty anchor on the header for backwards compatibility.

**Review by micalevisk — APPROVED** [review]
LGTM

I don't think that this should be backwards compatible

File: content/techniques/file-upload.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ uploadFile(files) {
 
 > info **Hint** The `FilesInterceptor()` decorator is exported from the `@nestjs/platform-express` package. The `@UploadedFiles()` decorator is exported from `@nestjs/common`.
 
-#### Multiple files
+#### Multiple fields
```
Comment (kamilmysliwiec) [inline content/techniques/file-upload.md:184]:
```suggestion
#### Multiple files
```

File: content/techniques/file-upload.md [hunk 2]
```diff
@@ -181,7 +181,7 @@ uploadFile(files) {
...
 
 To upload multiple fields (all with different field name keys), use the `FileFieldsInterceptor()` decorator. This decorator takes two arguments:
```
Comment (kamilmysliwiec) [inline content/techniques/file-upload.md:186]:
```suggestion
To upload multiple files (all with different field name keys), use the `FileFieldsInterceptor()` decorator. This decorator takes two arguments:
```

**Pull request merged**